app/src/main/kotlin/be/scri/activities/MainActivity.kt

```diff
@@ -116,7 +116,7 @@ class MainActivity : SimpleActivity(), GestureDetector.OnGestureListener {
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
```

Comment:
https://github.com/scribe-org/Scribe-Android/blob/abfea2235118948ec2573881cd37e660508f4a5a/app/src/main/kotlin/be/scri/activities/MainActivity.kt#L110-L124

These two functions can be removed as they are no longer required since the top menu has been removed.